content/docs/wiki/general/own-content.md

```diff
@@ -0,0 +1,28 @@
+This guide will help you replace the tonie content with your own content using teddycloud. 
+---
+
+Requirements:
+- Teddycloud running
+- Toniebox flashed
+- MP3 Audio file
+- Cloud enabled must be deactivated
+
+Assumptions:
+- Your Teddycloud IP is 192.168.1.1 
+
+---
+Guide:
+1. Open the web interface of teddycloud https://192.168.1.1/
```

Comment:
As the audio encoder is now also part of the new web gui, I would adapt it accordingly and just describe the new web gui process.